code/Form/UserFormsRequiredFieldsValidator.php

```diff
@@ -23,7 +23,7 @@
  *
  * @deprecated 5.4.0 Will be renamed to SilverStripe\UserForms\Form\UserFormsRequiredFieldsValidator
```

Comment:
```suggestion
```
Please also check for any other deprecation phpdoc or notices that snuck in

Reply:
Updated